app/views/grades/_review_table.html.erb

```diff
@@ -4,7 +4,12 @@
   <%end%>
 	<tr><th style="text-align:center;">&nbsp;</th>
 		<% for i in 0...reviews.size%>
-			<th style="text-align:center;"><%= collabel %>&nbsp;<%=i+1%>&nbsp;</th>
+      <% user = User.find_by_id(AssignmentParticipant.find(ResponseMap.find(reviews[i].map_id).reviewer_id).user_id) %>
+        <% if(user.role.instructor? || user.role.ta?) %>
```

Comment:
Actually, it is a design problem in Expertiza. Currently, `role_id` is in `User` table. In this case, if one student is TA of one assignment, s/he will be displayed as TA in all assignments.
 It will be better to put this field in `Participant` table.